### ISSUE TITLE
Control RabbitMQ prefetch settings from the config

### ISSUE THREAD

**Issue by Neodork** [sonata-project/SonataNotificationBundle#232]
## Subject
I've been using SonataNotificationBundle with RabbitMQ. I am running 2 queues under the same type "example_type". However I am missing the option to control the prefetch settings. I think it would be wise to add something to the configuration to change the prefetch settings.

So as explained by [RabbitMQ under the Fair dispatch chapter](https://www.rabbitmq.com/tutorials/tutorial-two-php.html) a queue can receive heavy and less heavy tasks (messages). One of my consumers is receiving a very heavy task and is busy quite some time depending on the amount of data to process. The other should then continue to process tasks but because prefetch is not set to 1 some tasks are not processed. For more information see "Steps to Reproduce".

## Steps to reproduce
We have Task A, B and C.
A = Long task
B = Short task
C = Short task

We have 2 consumers running:
D and E

Queue all messages in order A, B, C.

Consumer D shall receive Task A from RabbitMQ.
Consumer E shall receive Task B from RabbitMQ.
Consumer D shall receive Task C from RabbitMQ.

Consumer E finishes before D. But because Task C is assigned to Consumer D instead of being still in queue Consumer E is now not doing anything. Consumer E should however process Task C.

## Expected results
Consumer E not being idle because message was assigned to a busy consumer.

## Actual results
Consumer E is idle. Consumer D processes Task C after task A.

**TL;DR:**  There should be an option in the config to control the prefetch settings (in case of RabbitMQ).

**Comment by OskarStark** [comment]
@Neodork could you please verify #257 ?

**Comment by Neodork** [comment]
@OskarStark I will test this in production this month. Sorry for delayed answer.
